.github/workflows/release.yml

```diff
@@ -22,21 +21,25 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: npm
-      - name: install
-        run: npm ci || npm install
-      - run: npm test
-      - uses: fregante/daily-version-action@v1
+      - run: npm ci
+      - name: Test and build
+        run: npm test
+      - uses: fregante/daily-version-action@v2
         name: Create tag if necessary
-        id: daily-version
-      - if: steps.daily-version.outputs.created
+      - name: Update extension’s meta
+        run: npx dot-json@1 $DIRECTORY/manifest.json version $DAILY_VERSION
+      - uses: actions/upload-artifact@v3
+        with:
+          name: built
+          path: ${{ env.DIRECTORY }}
+      - if: env.DAILY_VERSION_CREATED
```

Comment:
This doesn't exist so the tag is created by `fregante/daily-version-action` but the release isn't

See:

- https://github.com/fregante/daily-version-action/pull/24
- https://github.com/fregante/daily-version-action/commit/46c5eeb36110c3cd7f169192aa4980cc7c266412